.travis.yml

```diff
@@ -12,12 +12,14 @@ addons:
     - google-chrome-stable
     - g++-4.8
 before_script:
-- npm install -g bower
+- npm install -g bower gulp-cli
```

Comment:
i have learned the hard way it is worth locking these installs via gulp-cli@<VERSION> :)